apps/src/code-studio/pd/workshop_survey/VariableFormGroup.jsx

```diff
@@ -205,16 +206,32 @@ const VariableFormGroup = React.createClass({
       return (<FormGroup />);
     }
 
-    const columnQuestions = this.props.columnVariableQuestions.map(question => (
-      <ColumnVariableQuestion
-        key={question.name}
-        question={question}
-        selectedValues={this.state.selected}
-        data={this.props.data}
-        errors={this.props.errors}
-        onChange={this.props.onChange}
-      />
-    ));
+    let columnQuestions;
+    if (this.state.selected.length < 1) {
```

Comment:
there's a built-in early exit [here](https://github.com/code-dot-org/code-dot-org/pull/15618/files#diff-720be733a75b36f093cb36d24f025e08R203) for that case